simulators/mujoco/mypy.ini

```diff
@@ -13,3 +13,6 @@ ignore_missing_imports = True
 
 [mypy-dm_control.*]
 ignore_missing_imports = True
+
+[mypy-cv2.*]
+ignore_missing_imports = True
```

Comment:
On my end this is not needed. With a clean virtual environment. Previously this was needed but iirc a recent CV update solved the issues.

Reply:
if you resolve but don't mak changes, can you explain to me why, then i understand the reason for leaving it as is

Reply:
bump